smac/facade/smac_ac_facade.py

```diff
@@ -427,6 +427,9 @@ def __init__(self,
             'min_chall': scenario.intens_min_chall  # type: ignore[attr-defined] # noqa F821
         }
 
+        if intensifier is None:
+            intensifier = Intensifier
+
         if isinstance(intensifier, Intensifier) \
```

Comment:
if intensifier is an instance, why do we need to define intensifier_def_kwargs? @dengdifan @renesass 
Or does it not matter because it does not hurt?

Reply:
I have modified the code to only define `intensifier_def_kwargs` when `intensifier` is a class.